src/NovelAudioService.cpp

```diff
@@ -155,29 +142,29 @@ void NovelAudioService::setSoundPosition(std::string soundName, int angle, int d
   }
   else
   {
-    std::cerr << "ERROR: " <<soundName << " is not allocated to a channel already!" << std::endl;
+    _logger.logErrorLine(soundName + " is not allocated to a channel already!");
   }
 }
 
 void NovelAudioService::setSoundDistance(std::string soundName, int distance) {
-    if (_channelMap.find(soundName) != _channelMap.end() && _channelMap[soundName] != MIXER_NO_EXPLICIT_CHANNEL)
+  if (_channelMap.find(soundName) != _channelMap.end() && _channelMap[soundName] != MIXER_NO_EXPLICIT_CHANNEL)
   {
     Mix_SetDistance(_channelMap[soundName], distance);
   }
   else
   {
-    std::cerr << "ERROR: " <<soundName << " is not allocated to a channel already!" << std::endl;
+    _logger.logErrorLine(soundName + " is not allocated to a channel already!");
   }
 }
 
 void NovelAudioService::setSoundPanning(std::string soundName, int leftChannelVolume, int rightChannelVolume) {
-    if (_channelMap.find(soundName) != _channelMap.end() && _channelMap[soundName] != MIXER_NO_EXPLICIT_CHANNEL)
+  if (_channelMap.find(soundName) != _channelMap.end() && _channelMap[soundName] != MIXER_NO_EXPLICIT_CHANNEL)
   {
     Mix_SetPanning(_channelMap[soundName], leftChannelVolume, rightChannelVolume);
   }
   else
   {
-    std::cerr << "ERROR: " <<soundName << " is not allocated to a channel already!" << std::endl;
+    _logger.logErrorLine(soundName + " is not allocated to a channel already!");
```

Comment:
Why is this concatenating with the new logger API?

Reply:
In certain cases, we can try to use `.append()` to reduce concatenation, however if we can't alter the string and we _desire_ a one line message, this is one of the only ways to pull it off with the current setup.

In any case, can I resolve this? I have changed the messaging to avoid this instead.

Reply:
Resolving.